.github/workflows/tests.yaml

```diff
@@ -44,8 +53,8 @@ jobs:
         # run: [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20]
 
     env:
-      TEST_ID: ${{ matrix.os }}-${{ matrix.python-version }}-${{ matrix.partition-label }}
-      # TEST_ID: ${{ matrix.os }}-${{ matrix.python-version }}-${{ matrix.partition-label }}-${{ matrix.run }}
+      TEST_ID: ${{ matrix.os }}-${{ matrix.python-version }}-${{ matrix.partition-label }}-${{ matrix.queuing }}
```

Comment:
I think this renaming will break some logic in our test report generation.

https://github.com/dask/distributed/blob/acf607832c7191cc496a9b4a81760170de85062c/continuous_integration/scripts/test_report.py#L146-L152
https://github.com/dask/distributed/blob/acf607832c7191cc496a9b4a81760170de85062c/continuous_integration/scripts/test_report.py#L353-L357

I introduced this in https://github.com/dask/distributed/pull/6837 

Worst case, we remove the more specific job URL again (we knew from the start this is very brittle)

Reply:
see https://github.com/dask/distributed/pull/6837/files#r939182529

Reply:
I don't think this should block this PR